curriculum/js1/8.js

```diff
@@ -5,8 +5,14 @@
  * @returns {number}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b, i=0, ans=0) => {
```

Comment:
You don't need an ans/res variable if you are not building up a result.
Since you are just checking every number i from 0 to a or b, just return i.